eclipse-collections-code-generator/src/main/resources/impl/map/mutable/primitiveObjectHashMap.stg

```diff
@@ -2454,17 +2454,18 @@ public class <name>ObjectHashMap\<V> implements Mutable<name>ObjectMap\<V>, Exte
             if (this.sentinelValues == null)
             {
                 this.sentinelValues = new SentinelValues\<>();
```

Comment:
in this case, it's better to invoke the function before mutating `this.sentinelValues`. Something like:
```
SentinelValues s = new SentinelValues();
s.zeroValue = function.valueOf(factory.value());
s.containsZeroKey = true;
this.sentinelValues = s;
```

and similarly below